### PR TITLE
Make CET and SMAP apply in more CPUs

### PR DESCRIPTION
**Pull request by superm1**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [x] Feature
- [ ] Documentation

### CODE REVIEW

File: docs/hsi-tests.d/org.fwupd.hsi.IntelCet.Active.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "id": "org.fwupd.hsi.IntelCet.Active",
-  "name": "Intel CET: Active",
+  "name": "CET Utilized by OS",
```
Comment (hughsie) [inline docs/hsi-tests.d/org.fwupd.hsi.IntelCet.Active.json:3]: I don't like the `Utilized` word -- maybe `CET In Use`? or `CET OS Support`?

File: plugins/cpu/fu-cpu-device.c [hunk 2]
```diff
@@ -310,14 +314,14 @@ fu_cpu_device_set_quirk_kv(FuDevice *device, const gchar *key, const gchar *valu
...
-fu_cpu_device_add_security_attrs_intel_cet_enabled(FuCpuDevice *self, FuSecurityAttrs *attrs)
+fu_cpu_device_add_security_attrs_cet_enabled(FuCpuDevice *self, FuSecurityAttrs *attrs)
 {
 	g_autoptr(FwupdSecurityAttr) attr = NULL;
 
 	/* create attr */
 	attr =
 	    fu_device_security_attr_new(FU_DEVICE(self), FWUPD_SECURITY_ATTR_ID_INTEL_CET_ENABLED);
-	fwupd_security_attr_set_result_success(attr, FWUPD_SECURITY_ATTR_RESULT_ENABLED);
+	fwupd_security_attr_set_result_success(attr, FWUPD_SECURITY_ATTR_RESULT_SUPPORTED);
```
Comment (hughsie) [inline plugins/cpu/fu-cpu-device.c:324]: I think we have to **add** this as a success enum in the json file too.

File: docs/hsi-tests.d/org.fwupd.hsi.IntelCet.Active.json (hunk 1 above)
Comment (hughsie) [reply docs/hsi-tests.d/org.fwupd.hsi.IntelCet.Active.json:3]: Makes sense to me.

Comment (hughsie) [reply docs/hsi-tests.d/org.fwupd.hsi.IntelCet.Active.json:3]: or "CET Hardware Support" perhaps?

Comment (superm1) [reply docs/hsi-tests.d/org.fwupd.hsi.IntelCet.Active.json:3]: the reason I wanted to use platform is that it's not it's not impossible that CPU microcode or BIOS microcode influences the ability for it to work.  But if you think `hardware support` is better I'll swap to that.

Comment (hughsie) [reply docs/hsi-tests.d/org.fwupd.hsi.IntelCet.Active.json:3]: Platform is probably more correct, agreed.

File: docs/hsi-tests.d/org.fwupd.hsi.IntelCet.Enabled.json [hunk 3]
```diff
@@ -11,14 +11,11 @@
     "not-supported": "CET not supported"
   },
   "success-results": {
-    "enabled": "CET feature enabled by the platform"
+    "supported": "CET feature support by the platform"
```
Comment (hughsie) [inline docs/hsi-tests.d/org.fwupd.hsi.IntelCet.Enabled.json:14]: I think we need both here to avoid regressions with insights

Comment (superm1) [reply docs/hsi-tests.d/org.fwupd.hsi.IntelCet.Enabled.json]: fixed

**Comment by superm1** [comment]
I did split this up so that the attribute rename happens in the ABI bump PR.  So I think we can safely `needs-backport` this PR.  Thoughts?

**Review by hughsie — APPROVED** [review]

**Pull request closed without merging**